blocks/aem-asset-selector/aem-asset-selector-util.js

```diff
@@ -378,6 +378,56 @@ export async function renderAssetSelectorWithImsFlow(cfg) {
     hideTreeNav: true,
     runningInUnifiedShell: false,
     noWrap: true,
+    filterSchema: [
+      {
+        header: 'File Type',
+        groupKey: 'TopGroup',
+        fields: [
+          {
+            element: 'checkbox',
+            name: 'type',
+            defaultValue: ['image/*'],
+            readOnly: true,
+            options: [
+              {
+                label: 'Images',
+                value: 'image/*',
+              },
+            ],
+          },
+        ],
+      },
+      {
+        fields:
+              [
+                {
+                  element: 'checkbox',
+                  name: 'type',
+                  options: [
+                    {
+                      label: 'JPG',
+                      value: 'image/jpeg',
+                    },
+                    {
+                      label: 'PNG',
+                      value: 'image/png',
+                    },
+                    {
+                      label: 'TIFF',
+                      value: 'image/tiff',
+                    },
+                    {
+                      label: 'GIF',
```

Comment:
Do we really want gif? I know users rarely have animated ones, but still. Handling of animated gifs is poor.